crates/fj-viewer/src/graphics/mod.rs

```diff
@@ -3,9 +3,12 @@
 mod draw_config;
 mod drawables;
 mod geometries;
+mod model;
```

Comment:
I think the naming of this module and some of the types in it is not ideal. It makes sense when viewed in isolation, but in the larger context of Fornjot, it could be confusing. Someone might search for the code that draws CAD models, and then think they found it here.

It's not a critical issue, and I don't have any better ideas right now. Just making a note, hoping that someone will come with with something better at some point.